README.md

```diff
@@ -16,10 +16,10 @@ Prerequisites: [Node](https://nodejs.org/en/download/) plus [Yarn](https://class
 > clone/fork 🏗 scaffold-eth:
 
 ```bash
-git clone https://github.com/austintgriffith/scaffold-eth.git
+git clone git@github.com:trufflesuite/scaffold-eth.git
```

Comment:
There should be a note with the alternative https endpoint for those who haven't set up ssh credentials on github